contracts/src/everlong/EverlongAdmin.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IEverlongAdmin } from "../interfaces/IEverlongAdmin.sol";
+
+/// @author DELV
+/// @title EverlongAdmin
+/// @notice Permissioning for Everlong.
+/// @custom:disclaimer The language used in this code is for coding convenience
+///                    only, and is not intended to, and does not, have any
+///                    particular legal or regulatory significance.
+contract EverlongAdmin is IEverlongAdmin {
+    /// @inheritdoc IEverlongAdmin
+    address public admin;
```

Comment:
This is fine for now, but it might be easier long term to have an EverlongStorage contract. Just want to put the idea on your radar